### ISSUE TITLE
Include a default "amiberry.conf" file in release package

### ISSUE THREAD

**Issue by midwan** [midwan/amiberry#440]
Although it's automatically generated by Amiberry itself, perhaps it would be a good idea to include a default `amiberry.conf` file on releases, to make things easier for new users.

**Comment by GeorgeSpiggott** [comment]
I just did a reinstall and was slightly confused when it wasn't there. Solved now.

On a related note I have a suggestion. There are two additional .conf files (rp-a500 and rp a1200) which I believe Amiberry does not read, even if they are selected in retropie where they show up as alternate emulator options.

Is it possible create different versions of those to solve the NTSC/PAL problem. They could also be used to allow Amiberry to boot as a much faster Amiga for playing 3D games.

**Comment by HoraceAndTheSpider** [comment]
> I just did a reinstall and was slightly confused when it wasn't there. Solved now.
> 
> On a related note I have a suggestion. There are two additional .conf files (rp-a500 and rp a1200) which I believe Amiberry does not read, even if they are selected in retropie where they show up as alternate emulator options.
> 
> Is it possible create different versions of those to solve the NTSC/PAL problem. They could also be used to allow Amiberry to boot as a much faster Amiga for playing 3D games.

Those Configs are not part of the Amiberry package. they are provided by RetroPie for booting ADF files.  We would like to replace the need for them by including ADFs as part of the AutoBooter solution. It should be easy enough to set AGA or NTSC requirements from file names, but to have specific settings for (for example) 3D games means we would need to pre-scan an existing file set (probably TOSEC) and store settings specific to certain games, maybe in another XML file. This is timely for someone to maintain and therefore probably not going to happen.

**Comment by GeorgeSpiggott** [comment]
Having _NTSC in the filename to pick up reducing the screen height to 200 or so (216 is the sweet spot IIRC) would be a great way to resolve the NTSC/PAL fullscreen issue.

I probably spoke before I tested regarding 3D games. I tried Gloom on WHDLoad and the game seems to run at full speed with the highest graphics settings all with no modification. If I press F12 to check the stats it reports a 68020/A1200 style setup. I don't remember the A1200 being that fast but it works fine anyway. I'll see if I can get a working copy of Alien Breed 3D 1 or 2 or Gloom Deluxe to test this further.
